dist/index.js

```diff
@@ -41248,48 +41248,29 @@ const run = async () => {
   core.startGroup('Extracting action context');
 
   // Setup Runner Context
-  const envRunnerFilters = [
```

Comment:
Though we supported arrays, we only ever sent a single item through. Making this a single item cleans up some processing below.